### PR TITLE
Prac 07 feedback 

### PR DESCRIPTION
**Pull request by JD-113**
@Terry-War
Please check project management and provide feedback

### CODE REVIEW

File: prac_07/myguitars.py [hunk 1]
```diff
@@ -1,56 +1,34 @@
 import csv
 from prac_07.guitar import Guitar
```
Comment (Terry-War) [inline prac_07/myguitars.py:9]: Estimate for time taken for completing this task?

**Comment by Terry-War** [comment]
I've added some comments for you to consider, but last time i did a code review they didn't come through on the other side for the author of the code, so i will add the same overall comment here for you just in case.

Everything looks good, i couldn't see anything wrong with the code for any of the programs, other than a few of them missing the file name or time estimate for completion in the header. All in all good work.

File: prac_07/myguitars.py (hunk 1 above)
Comment (JD-113) [reply prac_07/myguitars.py:9]: Thanks i do need to put a header, but did we have to put time for this one? though it was only project management? which I forgot anyway, again thank you for this.

**Comment by JD-113** [comment]
Thanks for the review Terry much appreciated.

**Pull request merged**